modules/troubleshooting-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins-reference.adoc
+
+:_content-type: REFERENCE
+[id="troubleshooting-dynamic-plug-in_{context}"]
+= Troubleshooting your dynamic plug-in
+
+Refer to this list of troubleshooting tips if you run into issues loading your plug-in. 
+
+* You have enabled your plug-in in the console Operator configuration. Run the following command to check to check if you have enabled your plug-in is in the console Operator configuration and your plug-in name is the output.
++
+[source,terminal]
+====
+oc get console.operator.openshift.io cluster -o jsonpath='{.spec.plugins}'
```

Comment:
You can also look at the enabled plugins in the status card of the Home -> Overview page in the Administrator perspective. You might need to refresh your browser if the plugin was recently enabled.